enclave/crypto/mbedtls/init.c

```diff
@@ -24,3 +30,21 @@ int oe_is_symcrypt_engine_available()
 {
     return 0;
 }
+
+int oe_is_symcrypt_provider_available()
+{
+    return 0;
```

Comment:
Is the assumption here that symcrypt provider is always available?

Reply:
No, this function is defined here so that tests using mbedtls can be compiled. Returning 0 signifies that symcrypt is not available for mbedtls